mmdet/apis/test.py

```diff
@@ -52,9 +52,10 @@ def single_gpu_test(model,
 
         # encode mask results
         if isinstance(result, tuple):
-            bbox_results, mask_results = result
-            encoded_mask_results = encode_mask_results(mask_results)
-            result = bbox_results, encoded_mask_results
+            result = list(result)
+            result[1] = encode_mask_results(result[1])
+            # Not sure if this is needed.
+            result = tuple(result)
```

Comment:
Please decide if conversion to `tuple` is actually needed and remove the comment.

Reply:
decided to use tuple, since using list cause crash of test.py script, do not think that investigation is required here to make use  of list